practice/src/javascripts/constants/apiUrl.js

```diff
@@ -0,0 +1,6 @@
+const API = {
+  BASE_URL: 'https://663896844253a866a24e6014.mockapi.io/api',
```

Comment:
You should get this value from the .env file because it helps you load the API endpoint for multiple environments.

You won't put .env file to git.